src/en/getting-started.md

```diff
@@ -1,255 +1,323 @@
-be used only by Juju for internal management, and a 'default' model, which is
-ready for actual use.
+<tr>
 
-The following command shows the currently active controller, model, and user:
+<td align="center" valign="center" border-width="0px" >
+<img src="./media/jaas-kubernetes.png" alt="Canonical kubernetes bundle" />
+<br />
+Press the green Add to model button to select the solution.
+</td>
 
-```bash 
-juju whoami
-```
+<td align="center" valign="center" border-width="0px">
+<img src="./media/jaas-deploy-changes.png" alt="deploying changes in JAAS" />
+<br />
+Press the blue Deploy changes button to proceed with the changes you've made
+with your model.
+</td>
 
-In our example, the output should look like this:
+</tr>
 
-```no-highlight
-Controller:  lxd-test
-Model:       default
-User:        admin
-```
+</table>
 
-## Deploy applications
+After you press `Deploy changes` you can adjust your model name and choose a
+cloud. When you select a cloud, you will be guided through
+the process of entering credentials for it.
 
-Juju is now ready to deploy applications from among the hundreds included in
-the [Juju charm store][charm store]. It is a good idea to test your new model.
-How about a MediaWiki site?
+## Prepare your cloud credentials
 
-```bash
-juju deploy cs:bundle/mediawiki-single
-```
+We recommend that you use each public cloud's identity and access management
+(IAM) tool to generate a new set of credentials exclusively for use with JAAS.
+
+See [Cloud credentials][credentials] for more information, along with the
+following links for your specific cloud.
+
+[Amazon Web Services][aws-creds] [Microsoft Azure][azure-creds] [Google Compute Engine][gce-creds]
+
+## Deploy
+
+Click on `Deploy` to confirm your cloud information and build your model.
+
+Deploying takes a few minutes - in this period, instances are being
+created in the cloud, software is being installed, applications are being
+related to each other and configuration is being applied.
+
+<table width="500" border-width="0px" cellpadding="5">
+
+<tr>
+
+<td align="center" valign="center" border-width="0px" >
+<img src="./media/jaas-deploy-1.png" alt="description here" />
+<br />
+Deploying a bundle can take a few minutes...
+</td>
+
+<td align="center" valign="center" border-width="0px">
+<img src="./media/jaas-deploy-2.png" alt="description here" />
+<br />
+When complete, the application icons will turn grey.
+</td>
+
+</tr>
+
+</table>
+
+
+
+As the applications become operational, the colours in the model view will
+change to grey to indicate an idle state and the pending notices in the
+inspector on the left will disappear, to show that everything is working as
+expected.
+
+!!! Tip: You can check and manage existing models through the JAAS GUI by
+clicking on your username
+
+## Use the command line
+
+Juju can also be used from the command line. Models you've created in JAAS are
+also able to be operated against via the CLI and vice versa. In order to use
```

Comment:
this sounds clunky to me. How about, `Models created with JAAS can also be operated on from the CLI, and vice versa - models created on the CLI can be modified by the GUI.`